ansible/roles/vm_set/templates/sonic.xml.j2

```diff
@@ -22,6 +22,14 @@
     <model fallback='forbid'/>
     <topology sockets='1' dies='1' cores='3' threads='2'/>
   </cpu>
+{% elif dut_name in ('vlab-05', 'vlab-06') %}
```

Comment:
Logic by DUT name is not robust enough. It could be easily broken if dut_name is changed. Is there other more robust way?

Reply:
Modified